contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -183,6 +189,9 @@ public String getFieldName() {
       lastColumn = plugin.getConfig().getLastColumn();
       allTextMode = plugin.getConfig().getAllTextMode();
       sheetName = plugin.getConfig().getSheetName();
+      maxArraySize = plugin.getConfig().getMaxArraySize();
```

Comment:
@pjfanning Thanks for the update!   Can you update the `README` file in this folder to explain what these options do?  Do you have recommended values?